docs/.ipynb_checkpoints/watermark-checkpoint.ipynb

```diff
@@ -1,434 +0,0 @@
-{
```

Comment:
Oh sorry, I made a mistake, I thought those were file changes, not a deletion. It's fine, I will also remove it from my side in future commits. This file shouldn't have been there in the first place :P All good!